code/modules/cargo/expressconsole.dm

```diff
@@ -20,7 +20,7 @@
 	var/list/meme_pack_data
 	var/obj/item/supplypod_beacon/beacon //the linked supplypod beacon
 	var/area/landingzone = /area/quartermaster/storage //where we droppin boys
-	var/podID = POD_STANDARD //0 is your standard supply supplypod (requires dissassembly after landing), 1 is the bluespace supply pod (teleports out after landing)
+	var/podType = /obj/structure/closet/supplypod //0 is your standard supply supplypod (requires dissassembly after landing), 1 is the bluespace supply pod (teleports out after landing)

```

Comment:
Please don't merge with the old comment in place.

Reply:
oh yeah i havent done anything with comments yet

Reply:
you still haven't done anything with it

Reply:
>implying
fuck ok

Reply:
Isn't this still wrong?